UnitySetup/UnitySetup.psm1

```diff
@@ -49,24 +49,37 @@ class UnitySetupInstance {
     [string]$Path
 
     UnitySetupInstance([string]$path) {
-
+        $version = $null
```

Comment:
I get these errors when I attempt to import your branch:

```
At C:\Users\User\Source\UnitySetup.powershell\UnitySetup\UnitySetup.psm1:52 char:9
+         $version = $null
+         ~~~~~~~~
Cannot assign property, use '$this.Version'.

At C:\Users\User\Source\UnitySetup.powershell\UnitySetup\UnitySetup.psm1:62 char:21
+                     $version = [UnityVersion]$Matches[0]
+                     ~~~~~~~~
Cannot assign property, use '$this.Version'.

At C:\Users\User\Source\UnitySetup.powershell\UnitySetup\UnitySetup.psm1:73 char:17
+                 $version = [UnityVersion]$xmlDoc.'ivy-module'.info.un ...
+                 ~~~~~~~~
Cannot assign property, use '$this.Version'.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : MissingThis
 
import-module : The module to process 'UnitySetup', listed in field 'ModuleToProcess/RootModule' of module manifest 'C:\Users\User\Source\UnitySetup.powershell\UnitySetup\UnitySetup.psd1' was 
not processed because no valid module was found in any module directory.
At line:1 char:1
+ import-module .\UnitySetup\UnitySetup.psd1 -force 2> errors.txt
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (UnitySetup:String) [Import-Module], PSInvalidOperationException
    + FullyQualifiedErrorId : Modules_ModuleFileNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand